localStorageExtension.js

```diff
@@ -0,0 +1,54 @@
+(function(Scratch) {
+    'use strict';
+    if (!Scratch.extensions.unsandboxed) {
+        throw new Error('Box2D must be run unsandboxed');
+    }
+    class LocalStorageExt {
+      getInfo () {
+        return {
+            id: "localstorageextension",
+            name: "Local Storage",
+            blocks: [
+                {
+                    opcode: 'lse_load_reporter',
```

Comment:
you don't need to prefix them on your own; the VM automatically inserts the extension's ID first. So you can just call this `load` and the other one `set`